docs/fast_data/tutorials/low-code.mdx

```diff
@@ -26,348 +26,347 @@ To do that, in the console, you will:
 
-<TabItem value="low-code">
-To change the ER Schema Config Map, move to the page of the Real-Time Updater and click on the <i>ConfigMaps & Secrets</i> tab. After that, select the <i>er-schema</i> ConfigMap.
-Its content must be replaced with the following:
+### Topics
 
-```json {4,6,10} showLineNumbers
-{
-  "version": "1.0.0",
-  "config": {
-    "registry": {
-      "outgoing": {
-        "order": {
-          "conditions": {
-            "registry_to_order": {
-              "condition": {
-                "registry_id": "registry_id"
-              },
-              "oneToMany": true
-            }
-          }
-        }
-      },
-      "order": {
-        "outgoing": {
-          "registry": {
-            "conditions": {
-              "order_to_registry": {
-                "condition": {
-                  "registry_id": "registry_id"
-                },
-                "oneToMany": false
-              }
-            }
-          }
-        }
-      }
-    }
-  }
-}
-```
+At the bottom of the same page, we can also see the Kafka topics. There are two types of topics:
 
-In this example, we can see that the relationship goes from registry to order (from line 4 to line 16) and vice versa.
+* [Ingestion topic](/fast_data/inputs_and_outputs.md#ingestion-message):  will be used by the SoR microservice
+to receive the data from the Source Systems so that the Projections will be updated on the target Database;
+* [PR Update topic](/fast_data/inputs_and_outputs.md#projection-update-message): will be used to notify when a projection has been updated.
 
-The relationship is represented by the following two conditions:
-- _registry_id_ (from order): _registry_id_ (from the updated registry). The condition is _One to One_, meaning that one order is related to one and only one registry item.
-- _registry_id_ (from registry): _registry_id_ (from the updated order). The condition is _One to Many_, meaning that one registry might be related to multiple orders.
 
-The condition is used to generate a `where` condition inside a MongoDB query: in this case, the condition can be translated as
-"where the registry_id inside order projection equals the value of the registry_id field of the registry projection".
+![Projection Topics](img/05b-page-overview-projection-kafka.png)
 
-In the context of the Real-Time Updater, the only relationship needed is the one going _from order to registry_, because when an order is updated, we want to see if there is already a document inside
-the _registry_ Projections that shares the same _registry_id_. If so, that document will be used as a reference to update the related Single View.
+:::caution
+The application does not create Kafka topics for you. If you decide to use a different Kafka topic name, please be sure the topic already exists and it is fully operational.
+The Real-Time Updater, the service that receives the update from the source System, will crash at startup if there is a topic not defined in Kafka.
```

Comment:
If the Projection Store service have the same behaviour of the Real-Time updated I would generalize this sentece refering to a generic SoR Service.
If the two services have different behaviours I would specify what happen if the Projection Storer is used insted of the Real-Time updater.

Reply:
they have the same behavior